lib/ros/be/infra/files/terraform/aws/s3/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "tags" {
+  type        = map(string)
+  default     = {}
+  description = "Tags for the created resources"
+}
+
+variable "aws_profile" {
+  type    = string
+  default = "default"
+}
+
```

Comment:
Same as before, no need to specify aws_profile

Reply:
fixed